inc/template-tags.php

```diff
@@ -211,6 +211,26 @@ function siteorigin_unwind_litespeed_lazy_exclude( $attr ) {
 	add_filter( 'siteorigin_unwind_logo_attributes', 'siteorigin_unwind_litespeed_lazy_exclude' );
 endif;
 
+if ( class_exists( 'Smush\WP_Smush' ) ) :
+	if ( ! function_exists( 'siteorigin_unwind_smush_lazy_exclude' ) ) :
+		/**
+		 * Exclude Logo from LiteSpeed Cache Lazy Load
+		 */
+		function siteorigin_unwind_smush_lazy_exclude( $attr, $attachment ) {
+			$custom_logo_id = siteorigin_setting( 'branding_logo' );
+			if ( empty( $custom_logo_id ) ) {
+				$custom_logo_id = get_theme_mod( 'custom_logo' );
+			}
+			
+			if ( ! empty( $custom_logo_id ) && $attachment->ID == $custom_logo_id ) {
+				$attr['class'] = ' no-lazyload';
```

Comment:
Do we expect other classes to be added to the logo image? Here is how it looks on my end.

<img width="1128" alt="SiteOrigin_–_Just_another_WordPress_site" src="https://user-images.githubusercontent.com/789159/67671367-7d9ac000-f97e-11e9-991b-d99b0d1478e5.png">